### PR TITLE
changed logic for displaying 'Show artist insights' button to account for minimum exhibitions

### PR DESCRIPTION
**Pull request by lilyfromseattle**
If an artist doesn't have artist insights, don't show the button
https://cl.ly/3dbfd53503a3
https://www.artsy.net/artwork/suzanne-heintz-the-tree

The "Show artist insights" button was displaying when the MarketInsights and SelectedExhibitions were not for the above artist and possible others.

It turned out that the artist in question did have an exhibition, but the SelectedExhibitions component had a minimum number of exhibitions (2) or it wouldn't render. I adjusted the logic controlling the "Show artist insights" button so that it wouldn't render if the number of exhibitions did not meet the minimum.

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 35c61c11ba7c295c918f06cc4619c1279db3ad09

https://deploy-preview-2342--artsy-reaction.netlify.com

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 0dba360fde31771308a81290e9de54bb20cf6f09

https://deploy-preview-2342--artsy-reaction.netlify.com

File: src/Apps/Artwork/Components/ArtistInfo.tsx [hunk 1]
```diff
@@ -97,7 +97,7 @@ export class ArtistInfo extends Component<ArtistInfoProps, ArtistInfoState> {
     const imageUrl = get(this.props, p => image.cropped.url)
     const showArtistInsightsButton =
       (artist.exhibition_highlights &&
-        artist.exhibition_highlights.length > 0) ||
+        artist.exhibition_highlights.length > 1) ||
```
Comment (ds300) [inline src/Apps/Artwork/Components/ArtistInfo.tsx]: Would be cool to keep this synched with the Selected Exhibitions component: https://github.com/ds300/reaction/blob/0eb9877c357fde2edb85091dfb3cc1e670d5bae4/src/Components/v2/SelectedExhibitions.tsx#L10

and then make it

```suggestion
        artist.exhibition_highlights.length >= MIN_EXHIBITIONS)
```

Comment (lilyfromseattle) [reply src/Apps/Artwork/Components/ArtistInfo.tsx]: How should I do that?

Comment (ds300) [reply src/Apps/Artwork/Components/ArtistInfo.tsx]: export `MIN_EXHIBITIONS` from the file I linked to above, `SelectedExhibitions.tsx`, then import it in this file and try out the suggested change above

**Comment by artsyit** [comment]
:rocket: PR was released in v16.6.3 :rocket:

**Pull request merged**